server/lib/genome/normalizers/drug_type.rb

```diff
@@ -17,58 +16,73 @@ def self.normalize_types
       end
 
-      def self.immunotherapy_drugs
-        DataModel::Drug.includes(:drug_attributes).where('drug_attributes.value' => immunotherapy_type_names)
+      private_class_method def self.immunotherapy_drugs
+        Drug.includes(:drug_attributes).where('drug_attributes.value' => immunotherapy_type_names)
       end
 
-      def self.immunotherapy_type_names
+      private_class_method def self.immunotherapy_type_names
         [
-            "immunosuppressive agents",
-            "immunosuppressant",
-            "immunomodulatory agents",
-            "immunomodulatory agent",
-            "immunostimulant",
-            "immunosuppressive agent"
+          'Antibody',
+          'Antibody-Drug Conjugate',
+          'Bovine Polyclonal Antibody',
+          'Chimeric antibody',
+          'Dna Based Immuno Therapy',
+          'Immunomodulatory Agent',
+          'Immunomodulatory Agents',
+          'Immunostimulant',
+          'Immunosuppressant',
+          'Immunosuppressive Agent',
+          'Immunosuppressive Agents',
+          'Immunosupressive agents',
+          'Immunotherapies',
+          'Monoclonal Antibody',
+          'Monoclonal Antibody',
+          'Therapeutic Antibodies',
+          'antiinflammatory agent,immunosuppressant'
         ]
       end
 
-      def self.antineoplastic_type_names
+      private_class_method def self.antineoplastic_type_names
         [
-          'antineoplastic agents',
-          'antineoplastic agents, protein kinase inhibitors',
-          'antineoplastic adjuncts',
-          'antineoplastic agents, hormonal',
-          'antineoplastic agents, homeopathic agents',
-          'antineoplastic agent',
-          'antineoplastic agents',
-          'antineoplastic agent',
-          'antineoplastic agents, phytogenic',
-          'anticancer agents',
-          'antineoplastic',
-          'anticarcinogenic agents',
-          'antineoplastics'
+          'Anticancer Agents',
+          'Anticarcinogenic Agents',
+          'Antineoplastic Adjuncts',
+          'Antineoplastic Agent',
+          'Antineoplastic Agents',
+          'Antineoplastic Agents',
+          'Antineoplastic Agents, Homeopathic Agents',
+          'Antineoplastic Agents, Hormonal',
+          'Antineoplastic Agents, Phytogenic',
+          'Antineoplastic Agents, Protein Kinase Inhibitors',
+          'Antineoplastic',
+          'Antineoplastics'
         ]
       end
 
-      def self.antineoplastic_source_names
+      private_class_method def self.antineoplastic_source_names
         [
-          'TALC',
-          'MyCancerGenome',
-          'MyCancerGenomeClinicalTrial',
+          'CGI',
+          'CKB',
           'CancerCommons',
+          'CIViC',
           'ClearityFoundationBiomarkers',
           'ClearityFoundationClinicalTrial',
-          'CKB',
+          'DoCM',
+          'MyCancerGenome',
+          'MyCancerGenomeClinicalTrial',
           'OncoKB',
-          'CGI'
+          'TALC'
```

Comment:
I rearranged the order so it looks like there are more changes, but the new sources here would be DoCM and CIViC